libraries/RC_Channel/RC_Channel.h

```diff
@@ -422,6 +423,8 @@ class RC_Channel {
 
     static const LookupTable lookuptable[];
 #endif
+
+    enum MAV_MOUNT_MODE _mount_mode_last = MAV_MOUNT_MODE_RETRACT;
```

Comment:
This state should live in the mount lib. That will remove the need to include MAVLink and get it past CI.

Reply:
really??? this will essentially mean that the complete switch logic is moved to the mount lib, which one can argue is equally illogic. Note that it is not the last mode from the viewpoint of the mount lib but relates to how the switches went (hence the if( > NEUTRAL) things).

Reply:
I'm not sure if this is admissibe and hence did not do (but did in my own 'original' code): Originally I simply used a static in the respective place in the .cpp file. The .cpp file has to include mavlink in any case due to the many usages of GCS_SEND_TXT. With this the include of mavink in the .h file would not be needed.
Would this do the trick and/or would be ok?

Reply:
on this case; go for a static for now . it prevent the mavlink include and use less flash ...